molecule/docker/tests/test_default.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+import os
+import yaml
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+@pytest.fixture()
+def AnsibleDefaults(Ansible):
+    with open("../../defaults/main.yml", 'r') as stream:
+        return yaml.load(stream)
+
+
+@pytest.mark.parametrize("dirs", [
+    "/etc/prometheus",
+    "/etc/prometheus/console_libraries",
+    "/etc/prometheus/consoles",
+    "/etc/prometheus/rules",
+    "/etc/prometheus/file_sd",
+    "/var/lib/prometheus"
+])
+def test_directories(host, dirs):
+    d = host.file(dirs)
+    assert d.is_directory
+    assert d.exists
+
+
+@pytest.mark.parametrize("files", [
+    "/etc/prometheus/prometheus.yml",
+    "/etc/prometheus/console_libraries/prom.lib",
+    "/etc/prometheus/consoles/prometheus.html",
+    "/etc/systemd/system/prometheus.service",
+    "/usr/local/bin/prometheus",
+    "/usr/local/bin/promtool"
+])
+def test_files(host, files):
+    f = host.file(files)
+    assert f.exists
+    assert f.is_file
+
+
+@pytest.mark.parametrize("files", [
```

Comment:
This should match directories set in `molecule/docker/playbook.yml` with variables such as `prometheus_db_dir` and `prometheus_config_dir`